src/screens/Auth/Register.tsx

```diff
@@ -142,21 +160,30 @@ const Register: React.FC<LoginProps> = ({ navigation }) => {
                 )}
                 name="passwordConfirm"
               />
-              <Text className="text-red-500">
+              <Text className="pb-4 text-red-500">
                 {errors.passwordConfirm &&
                   (errors.passwordConfirm.type === 'required'
                     ? 'This is required.'
                     : 'Passwords do not match.')}
               </Text>
             </View>
 
-            <View>
+            <View className="items-center">
               <Pressable
                 accessibilityRole="button"
-                className="my-2 w-20 rounded bg-blue-400 p-2"
+                className="w-60 rounded-lg bg-red-500 p-2"
                 onPress={handleSubmit((data) => onSubmit(data))}
               >
-                <Text className="text-center text-white">Submit</Text>
+                <Text
+                  style={{
+                    color: 'white',
+                    textAlign: 'center',
```

Comment:
You can use Tailwind instead of these native styles. Same for font size. The only one you should have to set without Tailwind is `fontFamily`.